maven can not build in mac OS, build error [ERROR] Unable to determine Groovy version.

I can not build from master branch, when run man build, it show [[ERROR] Unable to determine Groovy version].
$ sw_vers
```
ProductName:	Mac OS X
ProductVersion:	10.15.6
BuildVersion:	19G73
```
$ java -version
```
openjdk version "14.0.1" 2020-04-14
OpenJDK Runtime Environment (build 14.0.1+7)
OpenJDK 64-Bit Server VM (build 14.0.1+7, mixed mode, sharing)
```
$ mvn --version
```
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
Maven home: /Users/quorum/app/apache-maven-3.6.3
Java version: 14.0.1, vendor: Oracle Corporation, runtime: /Library/Java/JavaVirtualMachines/openjdk-14.0.1.jdk/Contents/Home
Default locale: en_JP, platform encoding: UTF-8
OS name: "mac os x", version: "10.15.6", arch: "x86_64", family: "mac"
```
$ groovy --version
```
Groovy Version: 3.0.4 JVM: 14.0.1 Vendor: Oracle Corporation OS: Mac OS X
```
$ git clone https://github.com/jpmorganchase/tessera.git
$ cd tessera
$ mvn install -Dmaven.test.skip=true

```

Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/groovy/groovy-all/2.1.5/groovy-all-2.1.5.jar (6.4 MB at 1.9 MB/s)
Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/groovy/groovy/2.5.6/groovy-2.5.6.jar (5.5 MB at 1.4 MB/s)
java.lang.NoClassDefFoundError: Could not initialize class org.codehaus.groovy.vmplugin.v7.Java7
	at org.codehaus.groovy.vmplugin.VMPluginFactory.<clinit>(VMPluginFactory.java:43)
	at org.codehaus.groovy.reflection.GroovyClassValueFactory.<clinit>(GroovyClassValueFactory.java:35)
	at org.codehaus.groovy.reflection.ClassInfo.<clinit>(ClassInfo.java:107)
	at org.codehaus.groovy.reflection.ReflectionCache.getCachedClass(ReflectionCache.java:95)
	at org.codehaus.groovy.reflection.ReflectionCache.<clinit>(ReflectionCache.java:39)
	at org.codehaus.groovy.runtime.metaclass.MetaClassRegistryImpl.registerMethods(MetaClassRegistryImpl.java:209)
	at org.codehaus.groovy.runtime.metaclass.MetaClassRegistryImpl.<init>(MetaClassRegistryImpl.java:107)
	at org.codehaus.groovy.runtime.metaclass.MetaClassRegistryImpl.<init>(MetaClassRegistryImpl.java:85)
	at groovy.lang.GroovySystem.<clinit>(GroovySystem.java:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.codehaus.gmaven.plugin.util.GroovyVersionHelper.getVersion(GroovyVersionHelper.java:84)
	at org.codehaus.gmaven.plugin.util.GroovyVersionHelper.detectVersion(GroovyVersionHelper.java:58)
	at org.codehaus.gmaven.plugin.RuntimeMojoSupport.ensureGroovyComparability(RuntimeMojoSupport.java:242)
	at org.codehaus.gmaven.plugin.RuntimeMojoSupport.prepare(RuntimeMojoSupport.java:173)
	at org.codehaus.gmaven.plugin.MojoSupport.execute(MojoSupport.java:44)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:210)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:156)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:148)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:305)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:957)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:289)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:193)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)
[ERROR] Unable to determine Groovy version
[WARNING] Ignoring null property: javax.persistence.jdbc.password
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for tessera 0.10.6-SNAPSHOT:
[INFO]
[INFO] tessera ............................................ SUCCESS [14:12 min]
[INFO] shared ............................................. SUCCESS [02:08 min]
[INFO] argon2 ............................................. SUCCESS [ 13.668 s]
[INFO] tests .............................................. SUCCESS [  0.780 s]
[INFO] test-util .......................................... SUCCESS [  1.955 s]
[INFO] service-locator .................................... SUCCESS [  0.807 s]
[INFO] service-locator-api ................................ SUCCESS [  0.985 s]
[INFO] encryption ......................................... SUCCESS [  0.680 s]
[INFO] encryption-api ..................................... SUCCESS [  7.293 s]
[INFO] encryption-jnacl ................................... SUCCESS [  2.847 s]
[INFO] encryption-ec ...................................... SUCCESS [  1.172 s]
[INFO] config ............................................. SUCCESS [ 11.253 s]
[INFO] cli ................................................ SUCCESS [  1.112 s]
[INFO] cli-api ............................................ SUCCESS [  2.470 s]
[INFO] security ........................................... SUCCESS [  2.687 s]
[INFO] server ............................................. SUCCESS [  0.842 s]
[INFO] server-api ......................................... SUCCESS [  1.254 s]
[INFO] server-utils ....................................... SUCCESS [  7.336 s]
[INFO] jersey-server ...................................... SUCCESS [ 12.029 s]
[INFO] jaxrs-client-unixsocket ............................ SUCCESS [  2.698 s]
[INFO] key-vault .......................................... SUCCESS [  0.954 s]
[INFO] key-vault-api ...................................... SUCCESS [  1.302 s]
[INFO] enclave ............................................ SUCCESS [  0.758 s]
[INFO] enclave-api ........................................ SUCCESS [  2.371 s]
[INFO] tessera-data ....................................... SUCCESS [ 10.388 s]
[INFO] tessera-context .................................... SUCCESS [  1.703 s]
[INFO] test-utils ......................................... SUCCESS [  0.726 s]
[INFO] mock-service-locator ............................... SUCCESS [  1.224 s]
[INFO] tessera-partyinfo .................................. SUCCESS [  3.277 s]
[INFO] tessera-core ....................................... SUCCESS [  2.134 s]
[INFO] tessera-jaxrs ...................................... SUCCESS [ 24.083 s]
[INFO] jaxrs-client ....................................... SUCCESS [  1.752 s]
[INFO] key-generation ..................................... SUCCESS [  1.505 s]
[INFO] config-cli ......................................... SUCCESS [  2.323 s]
[INFO] service-locator-spring ............................. SUCCESS [  1.140 s]
[INFO] common-jaxrs ....................................... SUCCESS [  2.637 s]
[INFO] tessera-dist ....................................... SUCCESS [  0.823 s]
[INFO] tessera-launcher ................................... SUCCESS [  1.606 s]
[INFO] mock-jaxrs ......................................... SUCCESS [  1.475 s]
[INFO] sync-jaxrs ......................................... SUCCESS [01:58 min]
[INFO] transaction-jaxrs .................................. SUCCESS [  3.186 s]
[INFO] thirdparty-jaxrs ................................... SUCCESS [  2.575 s]
[INFO] enclave-server ..................................... SUCCESS [  1.709 s]
[INFO] enclave-jaxrs ...................................... SUCCESS [ 28.202 s]
[INFO] azure-key-vault .................................... SUCCESS [ 44.794 s]
[INFO] hashicorp-key-vault ................................ SUCCESS [  8.262 s]
[INFO] aws-key-vault ...................................... SUCCESS [ 21.229 s]
[INFO] tessera-app ........................................ SUCCESS [ 17.519 s]
[INFO] tessera-simple ..................................... SUCCESS [  6.276 s]
[INFO] config-migration ................................... SUCCESS [  4.540 s]
[INFO] ddls ............................................... FAILURE [ 43.380 s]
[INFO] acceptance-test .................................... SKIPPED
[INFO] jmeter-test ........................................ SKIPPED
[INFO] encryption-kalium .................................. SKIPPED
[INFO] data-migration ..................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  23:35 min
[INFO] Finished at: 2020-07-29T15:52:35+09:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.codehaus.gmaven:groovy-maven-plugin:2.1:execute (h2-generate) on project ddls: Execution h2-generate of goal org.codehaus.gmaven:groovy-maven-plugin:2.1:execute failed: A required class was missing while executing org.codehaus.gmaven:groovy-maven-plugin:2.1:execute: Could not initialize class groovy.lang.GroovySystem
[ERROR] -----------------------------------------------------
[ERROR] realm =    plugin>org.codehaus.gmaven:groovy-maven-plugin:2.1
[ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
[ERROR] urls[0] = file:/Users/quorum/.m2/repository/org/codehaus/gmaven/groovy-maven-plugin/2.1/groovy-maven-plugin-2.1.jar
[ERROR] urls[1] = file:/Volumes/SSD/quorum/tessera/tessera-core/target/tessera-core-0.10.6-SNAPSHOT.jar
[ERROR] urls[2] = file:/Volumes/SSD/quorum/tessera/tessera-data/target/tessera-data-0.10.6-SNAPSHOT.jar
[ERROR] urls[3] = file:/Users/quorum/.m2/repository/org/bouncycastle/bcprov-jdk15on/1.61/bcprov-jdk15on-1.61.jar
[ERROR] urls[4] = file:/Users/quorum/.m2/repository/com/zaxxer/HikariCP/3.2.0/HikariCP-3.2.0.jar
[ERROR] urls[5] = file:/Users/quorum/.m2/repository/org/eclipse/persistence/org.eclipse.persistence.jpa/2.7.7/org.eclipse.persistence.jpa-2.7.7.jar
[ERROR] urls[6] = file:/Users/quorum/.m2/repository/org/eclipse/persistence/jakarta.persistence/2.2.3/jakarta.persistence-2.2.3.jar
[ERROR] urls[7] = file:/Users/quorum/.m2/repository/org/eclipse/persistence/org.eclipse.persistence.asm/2.7.7/org.eclipse.persistence.asm-2.7.7.jar
[ERROR] urls[8] = file:/Users/quorum/.m2/repository/org/eclipse/persistence/org.eclipse.persistence.antlr/2.7.7/org.eclipse.persistence.antlr-2.7.7.jar
[ERROR] urls[9] = file:/Users/quorum/.m2/repository/org/eclipse/persistence/org.eclipse.persistence.jpa.jpql/2.7.7/org.eclipse.persistence.jpa.jpql-2.7.7.jar
[ERROR] urls[10] = file:/Users/quorum/.m2/repository/org/eclipse/persistence/org.eclipse.persistence.extension/2.7.7/org.eclipse.persistence.extension-2.7.7.jar
[ERROR] urls[11] = file:/Users/quorum/.m2/repository/io/swagger/swagger-annotations/1.5.4/swagger-annotations-1.5.4.jar
[ERROR] urls[12] = file:/Users/quorum/.m2/repository/javax/validation/validation-api/2.0.1.Final/validation-api-2.0.1.Final.jar
[ERROR] urls[13] = file:/Volumes/SSD/quorum/tessera/enclave/enclave-api/target/enclave-api-0.10.6-SNAPSHOT.jar
[ERROR] urls[14] = file:/Volumes/SSD/quorum/tessera/encryption/encryption-api/target/encryption-api-0.10.6-SNAPSHOT.jar
[ERROR] urls[15] = file:/Volumes/SSD/quorum/tessera/service-locator/service-locator-api/target/service-locator-api-0.10.6-SNAPSHOT.jar
[ERROR] urls[16] = file:/Volumes/SSD/quorum/tessera/shared/target/shared-0.10.6-SNAPSHOT.jar
[ERROR] urls[17] = file:/Volumes/SSD/quorum/tessera/tessera-partyinfo/target/tessera-partyinfo-0.10.6-SNAPSHOT.jar
[ERROR] urls[18] = file:/Volumes/SSD/quorum/tessera/tessera-context/target/tessera-context-0.10.6-SNAPSHOT.jar
[ERROR] urls[19] = file:/Users/quorum/.m2/repository/javax/ws/rs/javax.ws.rs-api/2.1/javax.ws.rs-api-2.1.jar
[ERROR] urls[20] = file:/Users/quorum/.m2/repository/org/hibernate/validator/hibernate-validator/6.0.2.Final/hibernate-validator-6.0.2.Final.jar
[ERROR] urls[21] = file:/Users/quorum/.m2/repository/org/jboss/logging/jboss-logging/3.3.0.Final/jboss-logging-3.3.0.Final.jar
[ERROR] urls[22] = file:/Users/quorum/.m2/repository/com/fasterxml/classmate/1.3.1/classmate-1.3.1.jar
[ERROR] urls[23] = file:/Volumes/SSD/quorum/tessera/config/target/config-0.10.6-SNAPSHOT.jar
[ERROR] urls[24] = file:/Users/quorum/.m2/repository/org/glassfish/javax.el/3.0.1-b10/javax.el-3.0.1-b10.jar
[ERROR] urls[25] = file:/Users/quorum/.m2/repository/org/glassfish/javax.json/1.1.2/javax.json-1.1.2.jar
[ERROR] urls[26] = file:/Users/quorum/.m2/repository/javax/json/javax.json-api/1.1.2/javax.json-api-1.1.2.jar
[ERROR] urls[27] = file:/Volumes/SSD/quorum/tessera/argon2/target/argon2-0.10.6-SNAPSHOT.jar
[ERROR] urls[28] = file:/Users/quorum/.m2/repository/de/mkammerer/argon2-jvm/2.5/argon2-jvm-2.5.jar
[ERROR] urls[29] = file:/Users/quorum/.m2/repository/net/java/dev/jna/jna/4.5.2/jna-4.5.2.jar
[ERROR] urls[30] = file:/Volumes/SSD/quorum/tessera/encryption/encryption-jnacl/target/encryption-jnacl-0.10.6-SNAPSHOT.jar
[ERROR] urls[31] = file:/Users/quorum/.m2/repository/eu/neilalexander/jnacl/1.0.0/jnacl-1.0.0.jar
[ERROR] urls[32] = file:/Volumes/SSD/quorum/tessera/encryption/encryption-ec/target/encryption-ec-0.10.6-SNAPSHOT.jar
[ERROR] urls[33] = file:/Users/quorum/.m2/repository/org/apache/commons/commons-lang3/3.7/commons-lang3-3.7.jar
[ERROR] urls[34] = file:/Users/quorum/.m2/repository/org/jasypt/jasypt/1.9.3/jasypt-1.9.3.jar
[ERROR] urls[35] = file:/Volumes/SSD/quorum/tessera/key-vault/key-vault-api/target/key-vault-api-0.10.6-SNAPSHOT.jar
[ERROR] urls[36] = file:/Users/quorum/.m2/repository/javax/transaction/javax.transaction-api/1.3/javax.transaction-api-1.3.jar
[ERROR] urls[37] = file:/Users/quorum/.m2/repository/org/bouncycastle/bcpkix-jdk15on/1.61/bcpkix-jdk15on-1.61.jar
[ERROR] urls[38] = file:/Users/quorum/.m2/repository/org/springframework/spring-orm/5.2.3.RELEASE/spring-orm-5.2.3.RELEASE.jar
[ERROR] urls[39] = file:/Users/quorum/.m2/repository/org/springframework/spring-beans/5.2.3.RELEASE/spring-beans-5.2.3.RELEASE.jar
[ERROR] urls[40] = file:/Users/quorum/.m2/repository/org/springframework/spring-core/5.2.3.RELEASE/spring-core-5.2.3.RELEASE.jar
[ERROR] urls[41] = file:/Users/quorum/.m2/repository/org/springframework/spring-jcl/5.2.3.RELEASE/spring-jcl-5.2.3.RELEASE.jar
[ERROR] urls[42] = file:/Users/quorum/.m2/repository/org/springframework/spring-jdbc/5.2.3.RELEASE/spring-jdbc-5.2.3.RELEASE.jar
[ERROR] urls[43] = file:/Users/quorum/.m2/repository/org/springframework/spring-tx/5.2.3.RELEASE/spring-tx-5.2.3.RELEASE.jar
[ERROR] urls[44] = file:/Users/quorum/.m2/repository/com/squareup/retrofit2/retrofit/2.5.0/retrofit-2.5.0.jar
[ERROR] urls[45] = file:/Users/quorum/.m2/repository/com/squareup/okhttp3/okhttp/3.12.0/okhttp-3.12.0.jar
[ERROR] urls[46] = file:/Users/quorum/.m2/repository/com/squareup/okio/okio/1.15.0/okio-1.15.0.jar
[ERROR] urls[47] = file:/Users/quorum/.m2/repository/com/google/code/gson/gson/2.8.1/gson-2.8.1.jar
[ERROR] urls[48] = file:/Users/quorum/.m2/repository/ch/qos/logback/logback-classic/1.2.3/logback-classic-1.2.3.jar
[ERROR] urls[49] = file:/Users/quorum/.m2/repository/ch/qos/logback/logback-core/1.2.3/logback-core-1.2.3.jar
[ERROR] urls[50] = file:/Users/quorum/.m2/repository/org/eclipse/persistence/org.eclipse.persistence.moxy/2.7.7/org.eclipse.persistence.moxy-2.7.7.jar
[ERROR] urls[51] = file:/Users/quorum/.m2/repository/org/eclipse/persistence/org.eclipse.persistence.core/2.7.7/org.eclipse.persistence.core-2.7.7.jar
[ERROR] urls[52] = file:/Users/quorum/.m2/repository/javax/xml/bind/jaxb-api/2.3.0/jaxb-api-2.3.0.jar
[ERROR] urls[53] = file:/Users/quorum/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.0/jaxb-runtime-2.3.0.jar
[ERROR] urls[54] = file:/Users/quorum/.m2/repository/org/glassfish/jaxb/jaxb-core/2.3.0/jaxb-core-2.3.0.jar
[ERROR] urls[55] = file:/Users/quorum/.m2/repository/org/glassfish/jaxb/txw2/2.3.0/txw2-2.3.0.jar
[ERROR] urls[56] = file:/Users/quorum/.m2/repository/com/sun/istack/istack-commons-runtime/3.0.5/istack-commons-runtime-3.0.5.jar
[ERROR] urls[57] = file:/Users/quorum/.m2/repository/org/jvnet/staxex/stax-ex/1.7.8/stax-ex-1.7.8.jar
[ERROR] urls[58] = file:/Users/quorum/.m2/repository/com/sun/xml/fastinfoset/FastInfoset/1.2.13/FastInfoset-1.2.13.jar
[ERROR] urls[59] = file:/Users/quorum/.m2/repository/javax/activation/javax.activation-api/1.2.0/javax.activation-api-1.2.0.jar
[ERROR] urls[60] = file:/Users/quorum/.m2/repository/javax/annotation/javax.annotation-api/1.3.2/javax.annotation-api-1.3.2.jar
[ERROR] urls[61] = file:/Users/quorum/.m2/repository/org/codehaus/groovy/groovy/2.5.6/groovy-2.5.6.jar
[ERROR] urls[62] = file:/Users/quorum/.m2/repository/org/xerial/sqlite-jdbc/3.23.1/sqlite-jdbc-3.23.1.jar
[ERROR] urls[63] = file:/Users/quorum/.m2/repository/com/h2database/h2/1.4.199/h2-1.4.199.jar
[ERROR] urls[64] = file:/Users/quorum/.m2/repository/org/hsqldb/hsqldb/2.4.1/hsqldb-2.4.1.jar
[ERROR] urls[65] = file:/Users/quorum/.m2/repository/org/sonatype/gossip/gossip-slf4j/1.8/gossip-slf4j-1.8.jar
[ERROR] urls[66] = file:/Users/quorum/.m2/repository/org/sonatype/gossip/gossip-core/1.8/gossip-core-1.8.jar
[ERROR] urls[67] = file:/Users/quorum/.m2/repository/org/sonatype/gossip/gossip-bootstrap/1.8/gossip-bootstrap-1.8.jar
[ERROR] urls[68] = file:/Users/quorum/.m2/repository/com/google/guava/guava/14.0.1/guava-14.0.1.jar
[ERROR] urls[69] = file:/Users/quorum/.m2/repository/com/google/code/findbugs/jsr305/2.0.1/jsr305-2.0.1.jar
[ERROR] urls[70] = file:/Users/quorum/.m2/repository/org/codehaus/plexus/plexus-interpolation/1.16/plexus-interpolation-1.16.jar
[ERROR] urls[71] = file:/Users/quorum/.m2/repository/org/eclipse/aether/aether-util/0.9.0.M2/aether-util-0.9.0.M2.jar
[ERROR] urls[72] = file:/Users/quorum/.m2/repository/org/codehaus/gmaven/gmaven-adapter-api/2.1/gmaven-adapter-api-2.1.jar
[ERROR] urls[73] = file:/Users/quorum/.m2/repository/org/codehaus/gmaven/gmaven-adapter-impl/2.1/gmaven-adapter-impl-2.1.jar
[ERROR] urls[74] = file:/Users/quorum/.m2/repository/org/codehaus/groovy/groovy-all/2.1.5/groovy-all-2.1.5.jar
[ERROR] urls[75] = file:/Users/quorum/.m2/repository/org/apache/ant/ant/1.8.4/ant-1.8.4.jar
[ERROR] urls[76] = file:/Users/quorum/.m2/repository/org/apache/ant/ant-launcher/1.8.4/ant-launcher-1.8.4.jar
[ERROR] urls[77] = file:/Users/quorum/.m2/repository/jline/jline/1.0/jline-1.0.jar
[ERROR] urls[78] = file:/Users/quorum/.m2/repository/org/codehaus/plexus/plexus-utils/1.1/plexus-utils-1.1.jar
[ERROR] Number of foreign imports: 1
[ERROR] import: Entry[import  from realm ClassRealm[project>com.jpmorgan.quorum:tessera:0.10.6-SNAPSHOT, parent: ClassRealm[maven.api, parent: null]]]
[ERROR]
[ERROR] -----------------------------------------------------
[ERROR]
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginContainerException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :ddls
```

@melowe  thanks, because of this [https://github.com/gradle/gradle/issues/10248](https://github.com/gradle/gradle/issues/10248)
the gradle not work for jdk14, I try to figure out.

```
$ ./gradlew clean build
Downloading https://services.gradle.org/distributions/gradle-6.0.1-bin.zip
.........10%.........20%.........30%.........40%.........50%.........60%.........70%.........80%.........90%.........100%

Welcome to Gradle 6.0.1!

Here are the highlights of this release:
 - Substantial improvements in dependency management, including
   - Publishing Gradle Module Metadata in addition to pom.xml
   - Advanced control of transitive versions
   - Support for optional features and dependencies
   - Rules to tweak published metadata
 - Support for Java 13
 - Faster incremental Java and Groovy compilation
 - New Zinc compiler for Scala
 - VS2019 support
 - Support for Gradle Enterprise plugin 3.0

For more details see https://docs.gradle.org/6.0.1/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)
java.lang.NoClassDefFoundError: Could not initialize class org.codehaus.groovy.vmplugin.v7.Java7
        at org.codehaus.groovy.vmplugin.VMPluginFactory.<clinit>(VMPluginFactory.java:43)
        at org.codehaus.groovy.reflection.GroovyClassValueFactory.<clinit>(GroovyClassValueFactory.java:35)
        at org.codehaus.groovy.reflection.ClassInfo.<clinit>(ClassInfo.java:107)
        at org.codehaus.groovy.reflection.ReflectionCache.getCachedClass(ReflectionCache.java:95)
        at org.codehaus.groovy.reflection.ReflectionCache.<clinit>(ReflectionCache.java:39)
        at org.codehaus.groovy.runtime.metaclass.MetaClassRegistryImpl.registerMethods(MetaClassRegistryImpl.java:210)
        at org.codehaus.groovy.runtime.metaclass.MetaClassRegistryImpl.<init>(MetaClassRegistryImpl.java:108)
        at org.codehaus.groovy.runtime.metaclass.MetaClassRegistryImpl.<init>(MetaClassRegistryImpl.java:86)
        at groovy.lang.GroovySystem.<clinit>(GroovySystem.java:36)
        at org.codehaus.groovy.runtime.InvokerHelper.<clinit>(InvokerHelper.java:86)
        at groovy.lang.GroovyObjectSupport.getDefaultMetaClass(GroovyObjectSupport.java:59)
        at groovy.lang.GroovyObjectSupport.<init>(GroovyObjectSupport.java:32)
        at org.gradle.internal.extensibility.DefaultExtraPropertiesExtension.<init>(DefaultExtraPropertiesExtension.java:29)
        at org.gradle.internal.extensibility.DefaultConvention.<init>(DefaultConvention.java:49)
        at org.gradle.internal.extensibility.ExtensibleDynamicObject.<init>(ExtensibleDynamicObject.java:60)
        at org.gradle.internal.instantiation.generator.MixInExtensibleDynamicObject.<init>(MixInExtensibleDynamicObject.java:35)
        at org.gradle.initialization.DefaultSettings_Decorated.getAsDynamicObject(Unknown Source)
        at org.gradle.initialization.SettingsFactory.createSettings(SettingsFactory.java:58)
        at org.gradle.initialization.ScriptEvaluatingSettingsProcessor.process(ScriptEvaluatingSettingsProcessor.java:61)
        at org.gradle.initialization.PropertiesLoadingSettingsProcessor.process(PropertiesLoadingSettingsProcessor.java:38)
        at org.gradle.initialization.SettingsEvaluatedCallbackFiringSettingsProcessor.process(SettingsEvaluatedCallbackFiringSettingsProcessor.java:34)
        at org.gradle.initialization.RootBuildCacheControllerSettingsProcessor.process(RootBuildCacheControllerSettingsProcessor.java:36)
        at org.gradle.initialization.BuildOperationSettingsProcessor$2.call(BuildOperationSettingsProcessor.java:50)
        at org.gradle.initialization.BuildOperationSettingsProcessor$2.call(BuildOperationSettingsProcessor.java:47)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:416)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:406)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$1.execute(DefaultBuildOperationExecutor.java:165)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:250)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:158)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:102)
        at org.gradle.internal.operations.DelegatingBuildOperationExecutor.call(DelegatingBuildOperationExecutor.java:36)
        at org.gradle.initialization.BuildOperationSettingsProcessor.process(BuildOperationSettingsProcessor.java:47)
        at org.gradle.initialization.DefaultSettingsLoader.findSettingsAndLoadIfAppropriate(DefaultSettingsLoader.java:102)
        at org.gradle.initialization.DefaultSettingsLoader.findAndLoadSettings(DefaultSettingsLoader.java:45)
        at org.gradle.initialization.SettingsAttachingSettingsLoader.findAndLoadSettings(SettingsAttachingSettingsLoader.java:35)
        at org.gradle.internal.composite.CommandLineIncludedBuildSettingsLoader.findAndLoadSettings(CommandLineIncludedBuildSettingsLoader.java:34)
        at org.gradle.internal.composite.ChildBuildRegisteringSettingsLoader.findAndLoadSettings(ChildBuildRegisteringSettingsLoader.java:52)
        at org.gradle.internal.composite.CompositeBuildSettingsLoader.findAndLoadSettings(CompositeBuildSettingsLoader.java:35)
        at org.gradle.initialization.DefaultSettingsPreparer.prepareSettings(DefaultSettingsPreparer.java:36)
        at org.gradle.initialization.BuildOperatingFiringSettingsPreparer$LoadBuild.doLoadBuild(BuildOperatingFiringSettingsPreparer.java:59)
        at org.gradle.initialization.BuildOperatingFiringSettingsPreparer$LoadBuild.run(BuildOperatingFiringSettingsPreparer.java:54)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:402)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:394)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$1.execute(DefaultBuildOperationExecutor.java:165)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:250)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:158)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:92)
        at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
        at org.gradle.initialization.BuildOperatingFiringSettingsPreparer.prepareSettings(BuildOperatingFiringSettingsPreparer.java:42)
        at org.gradle.initialization.DefaultGradleLauncher.prepareSettings(DefaultGradleLauncher.java:194)
        at org.gradle.initialization.DefaultGradleLauncher.doClassicBuildStages(DefaultGradleLauncher.java:138)
        at org.gradle.initialization.DefaultGradleLauncher.doBuildStages(DefaultGradleLauncher.java:130)
        at org.gradle.initialization.DefaultGradleLauncher.executeTasks(DefaultGradleLauncher.java:110)
        at org.gradle.internal.invocation.GradleBuildController$1.execute(GradleBuildController.java:60)
        at org.gradle.internal.invocation.GradleBuildController$1.execute(GradleBuildController.java:57)
        at org.gradle.internal.invocation.GradleBuildController$3.create(GradleBuildController.java:85)
        at org.gradle.internal.invocation.GradleBuildController$3.create(GradleBuildController.java:78)
        at org.gradle.internal.work.DefaultWorkerLeaseService.withLocks(DefaultWorkerLeaseService.java:189)
        at org.gradle.internal.work.StopShieldingWorkerLeaseService.withLocks(StopShieldingWorkerLeaseService.java:40)
        at org.gradle.internal.invocation.GradleBuildController.doBuild(GradleBuildController.java:78)
        at org.gradle.internal.invocation.GradleBuildController.run(GradleBuildController.java:57)
        at org.gradle.tooling.internal.provider.ExecuteBuildActionRunner.run(ExecuteBuildActionRunner.java:31)
        at org.gradle.launcher.exec.ChainingBuildActionRunner.run(ChainingBuildActionRunner.java:35)
        at org.gradle.launcher.exec.BuildOutcomeReportingBuildActionRunner.run(BuildOutcomeReportingBuildActionRunner.java:63)
        at org.gradle.tooling.internal.provider.ValidatingBuildActionRunner.run(ValidatingBuildActionRunner.java:32)
        at org.gradle.launcher.exec.BuildCompletionNotifyingBuildActionRunner.run(BuildCompletionNotifyingBuildActionRunner.java:39)
        at org.gradle.launcher.exec.RunAsBuildOperationBuildActionRunner$3.call(RunAsBuildOperationBuildActionRunner.java:51)
        at org.gradle.launcher.exec.RunAsBuildOperationBuildActionRunner$3.call(RunAsBuildOperationBuildActionRunner.java:45)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:416)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:406)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$1.execute(DefaultBuildOperationExecutor.java:165)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:250)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:158)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:102)
        at org.gradle.internal.operations.DelegatingBuildOperationExecutor.call(DelegatingBuildOperationExecutor.java:36)
        at org.gradle.launcher.exec.RunAsBuildOperationBuildActionRunner.run(RunAsBuildOperationBuildActionRunner.java:45)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter$1.transform(InProcessBuildActionExecuter.java:50)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter$1.transform(InProcessBuildActionExecuter.java:47)
        at org.gradle.composite.internal.DefaultRootBuildState.run(DefaultRootBuildState.java:78)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:47)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:31)
        at org.gradle.launcher.exec.BuildTreeScopeBuildActionExecuter.execute(BuildTreeScopeBuildActionExecuter.java:42)
        at org.gradle.launcher.exec.BuildTreeScopeBuildActionExecuter.execute(BuildTreeScopeBuildActionExecuter.java:28)
        at org.gradle.tooling.internal.provider.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:78)
        at org.gradle.tooling.internal.provider.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:52)
        at org.gradle.tooling.internal.provider.SubscribableBuildActionExecuter.execute(SubscribableBuildActionExecuter.java:59)
        at org.gradle.tooling.internal.provider.SubscribableBuildActionExecuter.execute(SubscribableBuildActionExecuter.java:36)
        at org.gradle.tooling.internal.provider.SessionScopeBuildActionExecuter.execute(SessionScopeBuildActionExecuter.java:68)
        at org.gradle.tooling.internal.provider.SessionScopeBuildActionExecuter.execute(SessionScopeBuildActionExecuter.java:38)
        at org.gradle.tooling.internal.provider.GradleThreadBuildActionExecuter.execute(GradleThreadBuildActionExecuter.java:37)
        at org.gradle.tooling.internal.provider.GradleThreadBuildActionExecuter.execute(GradleThreadBuildActionExecuter.java:26)
        at org.gradle.tooling.internal.provider.ParallelismConfigurationBuildActionExecuter.execute(ParallelismConfigurationBuildActionExecuter.java:43)
        at org.gradle.tooling.internal.provider.ParallelismConfigurationBuildActionExecuter.execute(ParallelismConfigurationBuildActionExecuter.java:29)
        at org.gradle.tooling.internal.provider.StartParamsValidatingActionExecuter.execute(StartParamsValidatingActionExecuter.java:60)
        at org.gradle.tooling.internal.provider.StartParamsValidatingActionExecuter.execute(StartParamsValidatingActionExecuter.java:32)
        at org.gradle.tooling.internal.provider.SessionFailureReportingActionExecuter.execute(SessionFailureReportingActionExecuter.java:55)
        at org.gradle.tooling.internal.provider.SessionFailureReportingActionExecuter.execute(SessionFailureReportingActionExecuter.java:41)
        at org.gradle.tooling.internal.provider.SetupLoggingActionExecuter.execute(SetupLoggingActionExecuter.java:48)
        at org.gradle.tooling.internal.provider.SetupLoggingActionExecuter.execute(SetupLoggingActionExecuter.java:32)
        at org.gradle.launcher.daemon.server.exec.ExecuteBuild.doBuild(ExecuteBuild.java:68)
        at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at org.gradle.launcher.daemon.server.exec.WatchForDisconnection.execute(WatchForDisconnection.java:39)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at org.gradle.launcher.daemon.server.exec.ResetDeprecationLogger.execute(ResetDeprecationLogger.java:27)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at org.gradle.launcher.daemon.server.exec.RequestStopIfSingleUsedDaemon.execute(RequestStopIfSingleUsedDaemon.java:35)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.create(ForwardClientInput.java:78)
        at org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.create(ForwardClientInput.java:75)
        at org.gradle.util.Swapper.swap(Swapper.java:38)
        at org.gradle.launcher.daemon.server.exec.ForwardClientInput.execute(ForwardClientInput.java:75)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at org.gradle.launcher.daemon.server.exec.LogAndCheckHealth.execute(LogAndCheckHealth.java:55)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at org.gradle.launcher.daemon.server.exec.LogToClient.doBuild(LogToClient.java:63)
        at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at org.gradle.launcher.daemon.server.exec.EstablishBuildEnvironment.doBuild(EstablishBuildEnvironment.java:82)
        at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at org.gradle.launcher.daemon.server.exec.StartBuildOrRespondWithBusy$1.run(StartBuildOrRespondWithBusy.java:52)
        at org.gradle.launcher.daemon.server.DaemonStateCoordinator$1.run(DaemonStateCoordinator.java:297)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
        at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
        at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:56)
        at java.base/java.lang.Thread.run(Thread.java:832)

FAILURE: Build failed with an exception.

* What went wrong:
Could not initialize class org.codehaus.groovy.reflection.ReflectionCache

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 18s

```

--- a/build.gradle
+++ b/build.gradle
@@ -29,10 +29,10 @@ allprojects {
 
                 testCompile "junit:junit:4.12"
                 testCompile "org.assertj:assertj-core:3.9.1"
-                testCompile "org.mockito:mockito-core:2.23.0"
+                testCompile "org.mockito:mockito-core:3.4.4"
                 testCompile "com.openpojo:openpojo:0.8.10"
                 testCompile "com.github.stefanbirkner:system-rules:1.18.0"
-                testCompile "nl.jqno.equalsverifier:equalsverifier:3.1.5"
+                testCompile "nl.jqno.equalsverifier:equalsverifier:3.4.1"
                 compile "com.github.jnr:jnr-unixsocket:0.28"
                 compile "de.mkammerer:argon2-jvm:2.5"
                 compile "javax.validation:validation-api:2.0.1.Final"

--- a/ddls/pom.xml
+++ b/ddls/pom.xml
@@ -30,7 +30,7 @@
                 <plugin>
                     <groupId>org.codehaus.gmaven</groupId>
                     <artifactId>groovy-maven-plugin</artifactId>
-                    <version>2.1</version>
+                    <version>2.1.1</version>
                 </plugin>
             </plugins>
         </pluginManagement>
@@ -52,7 +52,7 @@
                     <dependency>
                         <groupId>org.codehaus.groovy</groupId>
                         <artifactId>groovy</artifactId>
-                        <version>2.5.6</version>
+                        <version>3.0.5</version>
                     </dependency>
                     
                     <dependency>

--- a/pom.xml
+++ b/pom.xml
@@ -77,7 +77,7 @@
                 <plugin>
                     <groupId>org.jacoco</groupId>
                     <artifactId>jacoco-maven-plugin</artifactId>
-                    <version>0.8.3</version>
+                    <version>0.8.5</version>
                     <dependencies>
                         <dependency>
                             <groupId>commons-collections</groupId>
@@ -745,7 +745,7 @@
             <dependency>
                 <groupId>org.mockito</groupId>
                 <artifactId>mockito-core</artifactId>
-                <version>3.3.3</version>
+                <version>3.4.4</version>
                 <scope>test</scope>
             </dependency>
 
@@ -1024,7 +1024,7 @@
             <dependency>
                 <groupId>nl.jqno.equalsverifier</groupId>
                 <artifactId>equalsverifier</artifactId>
-                <version>3.1.5</version>
+                <version>3.4.1</version>
             </dependency>
 
             <dependency>
@@ -1417,7 +1417,7 @@
             <plugin>
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
-                <version>0.8.2</version>
+                <version>0.8.5</version>
 
                 <reportSets>
                     <reportSet>

